docs/managed-updates.md

```diff
@@ -1,4 +1,23 @@
-# Managed Updates (Alpha)
+# Managed Updates
+
+****** OBSOLETE ******
```

Comment:
Why obsolete?

Reply:
Looking at this again - it should probably be "Deprecated" rather than obsolete, because we aren't going to update this doc anymore.